docs/how-to-provision-using-dps.md

```diff
@@ -239,30 +249,38 @@ az functionapp config appsettings set --settings "AdtAppId=<Application (client)
 
 ### Create Device Provisioning enrollment
 
-Next, you'll need to create an enrollment in Device Provisioning Service using a custom allocation function. How to do this can be found in [Create the enrollment](../iot-dps/how-to-use-custom-allocation-policies#create-the-enrollment.md).
+Next, you'll need to create an enrollment in Device Provisioning Service using a custom allocation function. You can find the instructions to do this in [*Create the enrollment*](../iot-dps/how-to-use-custom-allocation-policies#create-the-enrollment.md).
 
-Link the enrollment to the function you just created, by selecting the function in the 'select how you want to assign devices to hubs' field. The enrollment name and primary or secondary SAS key will be used later to configure the device simulator.
+Link the enrollment to the function you just created, by selecting the function in the 'Select how you want to assign devices to hubs' field. The enrollment name and primary or secondary SAS key will be used later to configure the device simulator.
 
 ### Setting up the device simulator
 
-This sample uses a device simulator that includes provisioning using the Device Provisioning Service. The device simulator is located here: [Azure Digital Twins and IoT Hub Integration Sample](https://github.com/Azure-Samples/digital-twins-iothub-integration/tree/main/device-simulator). Get the sample project on your machine by navigating to the sample link root repo and selecting the Download ZIP button underneath the title.
+This sample uses a device simulator that includes provisioning using the Device Provisioning Service. The device simulator is located here: [Azure Digital Twins and IoT Hub Integration Sample](https://docs.microsoft.com/samples/azure-samples/digital-twins-iothub-integration/adt-iothub-provision-sample/). Get the sample project on your machine by navigating to the sample link and selecting the *Download ZIP* button underneath the title. Unzip the downloaded folder.
+
+The device simulator is based on Node.js, version 10.0.x or later. [*Prepare your development environment*](https://github.com/Azure/azure-iot-sdk-node/blob/master/doc/node-devbox-setup.md) describes how to install Node.js for this tutorial on either Windows or Linux.
```

Comment:
This could be part of the prerequisites section

Reply:
Agree. At least list it there, and then reference it here.